### ISSUE TITLE
p40 two gpu predict ok? but no-sound video is a black screen.

### ISSUE THREAD

**Issue by tim20160413** [Tencent-Hunyuan/HunyuanVideo-Avatar#60]
use p40 ，two gpu，command for this:
    torchrun --nnodes=1 --nproc_per_node=2 --master_port 29500  hymm_sp/sample_gpu_poor.py \
        --input 'assets/test.csv' \
        --ckpt ${checkpoint_path} \
        --sample-n-frames 64 \
        --seed 128 \
        --image-size 704 \
        --cfg-scale 6.0 \
        --infer-steps 30 \
        --use-deepcache 0 \
        --flow-shift-eval-video 2.5 \
        --save-path ${OUTPUT_BASEPATH} \
        --use-fp8 \
        --use-int8 \
        --cpu-offload \
        --infer-min

for supporting the fp8，Simple modification ：'bf16' ->  torch.float16，output file：*_audio.mp4，but it is  a black screen with backgroup music which is right feely。

find code ：samples = hunyuan_video_sampler.predict(args, batch, wav2vec, feature_extractor, align_instance)
samples saved the file: .mp4 which  is  a black screen ,  

What could be the reasons for the issue mentioned above? maybe

**Comment by tim20160413** [comment]
other points: 1. code has used 'initialize_distributed()' for multi-gpu ;  2. HunyuanVideoSampler uses "torch.nn.DataParallel()" for multi-gpu
